static/js/locate.js

```diff
@@ -0,0 +1,67 @@
+var map;
+function initialize() {
+    var myLatlng = new google.maps.LatLng(document.getElementById("lat").value, document.getElementById("long").value);
```

Comment:
We should not be hard-coding the ids of element here, it should be passed when initialized or something.

With the current way, we cannot have more that one map instances on a page. And we have to use `lat`, `long`, `map_canvas`, `my-address` exactly as it is.